azurerm/resource_arm_healthcare_service.go

```diff
@@ -0,0 +1,436 @@
+			if err := d.Set("authentication_configuration", flattenHealthcareAuthConfig(authConfig)); err != nil {
+				return fmt.Errorf("Error setting `authentication_configuration`: %+v", flattenHealthcareAuthConfig(authConfig))
+			}
+		}
+
+		if corsConfig := properties.CorsConfiguration; corsConfig != nil {
+			if err := d.Set("cors_configuration", flattenHealthcareCorsConfig(corsConfig)); err != nil {
+				return fmt.Errorf("Error setting `cors_configuration`: %+v", flattenHealthcareCorsConfig(corsConfig))
+			}
+		}
+	}
+
+	flattenAndSetTags(d, resp.Tags)
+
+	return nil
+}
+
+func resourceArmHealthcareServiceDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).Healthcare.HealthcareServiceClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := parseAzureResourceID(d.Id())
+	if err != nil {
+		return fmt.Errorf("Error Parsing Azure Resource ID: %+v", err)
+	}
+	resGroup := id.ResourceGroup
+	name := id.Path["services"]
+	future, err := client.Delete(ctx, resGroup, name)
+	if err != nil {
+		return fmt.Errorf("Error deleting Healthcare Service %q (Resource Group %q): %+v", name, resGroup, err)
+	}
+
+	if err = future.WaitForCompletionRef(ctx, client.Client); err != nil {
+		return fmt.Errorf("Error waiting for the deleting Healthcare Service %q (Resource Group %q): %+v", name, resGroup, err)
+	}
+
+	return nil
+}
+
+func expandAzureRMhealthcareapisAccessPolicyEntries(d *schema.ResourceData) *[]healthcareapis.ServiceAccessPolicyEntry {
+	accessPolicyObjectIds := d.Get("access_policy_object_ids").([]interface{})
+	svcAccessPolicyArray := make([]healthcareapis.ServiceAccessPolicyEntry, 0)
+
+	for _, objectId := range accessPolicyObjectIds {
+		objectIdsStr := objectId.(string)
+		svcAccessPolicyObjectId := healthcareapis.ServiceAccessPolicyEntry{ObjectID: &objectIdsStr}
+		svcAccessPolicyArray = append(svcAccessPolicyArray, svcAccessPolicyObjectId)
+	}
+
+	return &svcAccessPolicyArray
+}
+
+func expandAzureRMhealthcareapisCorsConfiguration(d *schema.ResourceData) *healthcareapis.ServiceCorsConfigurationInfo {
+	corsConfigRaw := d.Get("cors_configuration").([]interface{})
+
+	if len(corsConfigRaw) == 0 {
+		return &healthcareapis.ServiceCorsConfigurationInfo{}
+	}
+
+	allowedOrigins := make([]string, 0)
+	allowedHeaders := make([]string, 0)
+	allowedMethods := make([]string, 0)
+	maxAgeInSeconds := int32(0)
+	allowCredentials := true
+
+	/*for _, attr := range corsConfigRaw {
+		corsConfigAttr := attr.(map[string]interface{})
+
+		allowedOrigins = *utils.ExpandStringSlice(corsConfigAttr["allowed_origins"].([]interface{}))
+		allowedHeaders = *utils.ExpandStringSlice(corsConfigAttr["allowed_headers"].([]interface{}))
+		allowedMethods = *utils.ExpandStringSlice(corsConfigAttr["allowed_methods"].([]interface{}))
+		maxAgeInSeconds = int32(corsConfigAttr["max_age_in_seconds"].(int))
+		allowCredentials = corsConfigAttr["allow_credentials"].(bool)
+	}
+	*/
```

Comment:
if we don't need this we should remove the comment

Reply:
oh, sorry. Done